concent_api/core/tests/utils.py

```diff
@@ -616,3 +643,9 @@ def _assert_stored_message_counter_not_increased(self):
 
     def _assert_client_count_is_equal(self, count):
         self.assertEqual(Client.objects.count(), count)
+
+    def _create_payment_object(self, amount = None, closure_time = None):
+        payment_item = mock.Mock()
+        payment_item.amount         = amount
+        payment_item.closure_time   = closure_time
+        return payment_item
```

Comment:
please add new line at the end of the file ;)

Reply:
Done